tools/create_xbuildenv.py

```diff
@@ -120,8 +121,42 @@ def create(
         encoding="utf8",
     )
     if res.returncode != 0:
-        logger.error("Failed to run pip freeze:")
-        exit_with_stdio(res)
```

Comment:
Why not import `exit_with_stdio` from `pyodide_build`?

Reply:
I tried to remove dependencies on pyodide-build as much as possible, so that changes in pyodide-build would not break this script.